guide/index.html

```diff
@@ -0,0 +1,230 @@
+    <meta property="og:type" content="website" />
+    <meta property="og:description" content="Technology professionals using our skills, tools, and influence to amplify the Climate Movement" />
+
+    <!-- Twitter Metadata -->
+    <meta name="twitter:card" content="summary" />
+    <meta name="twitter:site" content="ClimateAction.tech" />
+    <meta name="twitter:title" content="ClimateAction.tech" />
+    <meta name="twitter:description" content="Technology professionals using our skills, tools, and influence to amplify the Climate Movement." />
+    <meta name="twitter:image" content="https://climateaction.tech/img/earth.png" />
+
+    <title>ClimateAction.tech</title>
+
+    <!-- Bootstrap Core CSS -->
+    <link href="/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
+
+    <!-- Theme CSS -->
+    <link href="/css/freelancer-3.css" rel="stylesheet">
+
+    <!-- Custom Fonts -->
+    <link href="vendor/font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic" rel="stylesheet" type="text/css">
+
+    <!-- Favicon -->
+    <link rel="icon" type="image/png" href="/img/earth.png" />
+
+    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+    <!--[if lt IE 9]>
+        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
+        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
+    <![endif]-->
+
+</head>
+
+<body id="page-top" class="index">
+<div id="skipnav"><a href="#maincontent">Skip to main content</a></div>
+
+    <!-- Navigation -->
+    <nav id="mainNav" class="navbar navbar-default navbar-fixed-top navbar-custom">
+        <div class="container">
+            <!-- Brand and toggle get grouped for better mobile display -->
+            <div class="navbar-header page-scroll">
+                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+                    <span class="sr-only">Toggle navigation</span> Menu <i class="fa fa-bars"></i>
+                </button>
+                <a class="navbar-brand" href="#page-top">ClimateAction.tech</a>
+            </div>
+
+            <!-- Collect the nav links, forms, and other content for toggling -->
+            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+                <ul class="nav navbar-nav navbar-right">
+                    <li class="page-scroll">
+                        <a href="#about">About</a>
```

Comment:
```suggestion
                        <a href="/#about">About</a>
```

Reply:
(and the other menu items as well)

Reply:
Ah good spot @melissahsiung. Also I notice you have done something similar in b9cbbcef0cfdc40ace48a4a9472ab6baf8abedb3

But we shouldn't put the `/` on `#page-top` or `#maincontent` as these IDs exist on both pages. They just help users navigate to the to top of the page / content.

I'll fix this now in my PR

Reply:
OK I think I've got these covered here 223f079